src/style.css

```diff
@@ -0,0 +1,77 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf) format("truetype");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf) format("truetype");
+  font-style: bold;
+  font-weight: 900;
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.big-form,
+.small-form {
+  margin-top: 20px;
+}
+
+.form,
+.form--small {
+  box-sizing: border-box;
+  width: 100%;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  font-family: inherit;
+  color: #3d4e61;
+}
```

Comment:
No need to apply this styles for both classes. Create one class for form and then overlap sizes for small one

Big search bar - is a default element, and small searchbar - is a modified default element